Scooty/ScootyTests/AccountantTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Scooty;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Scooty.Tests
+{
+    [TestClass()]
+    public class AccountantTests
+    {
+        public RentalCompany test = new RentalCompany("test");
+
+        [TestMethod()]
+        public void CalcSessionPriceTest_inputShortScooterSession_returnPriceOfSession()
+        {
+            test.AddScooter("test", .10m);
+            var start = new DateTime(1, 1, 1, 20, 15, 00);
+            var end = new DateTime(1, 1, 1, 20, 28, 00);
+            var expect = 1.3m;
+
+            var result = test.accountant.CalcSessionPrice(test.GetScooterById("test"), start, end);
```

Comment:
Šis vairāk izskatās pēc Accountant testa nekā RentalCompany testa. 
jābūt tad atsevišķi.